template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/Views/Home/Clients.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@model IEnumerable<ClientViewModel>
+
+<h4>Clients</h4>
```

Comment:
There's quite a bit of variance. We should probably do a separate HTML consistency pass and general design pass.